container/build_container.sh

```diff
@@ -7,7 +7,8 @@ declare -A containers=(
     [kraken2]=https://depot.galaxyproject.org/singularity/kraken2:2.1.2--pl5321h9f5acd7_3
     [bracken]=https://depot.galaxyproject.org/singularity/bracken:2.8--py39hc16433a_0
     [samtools]=https://depot.galaxyproject.org/singularity/samtools:1.17--h00cdaf9_0
-    [ariba]=https://depot.galaxyproject.org/singularity/ariba:2.14.6--py39h8f06ca0_2
```

Comment:
There's a bunch of places that still mention ariba. Quick search showed the readme, stale references in config, deploy_reqs.sh. Probably want to either delete them or chuck them in a legacy/ repo.

Reply:
Will do!